client/reader/start/card.jsx

```diff
@@ -51,10 +51,11 @@ const StartCard = React.createClass( {
 				'is-photo': hasPost && post.excerpt.length < 1
 			}
 		);
+		const railcar = get( this.props.recommendation, 'railcar' );
 
 		return (
 			<Card className={ cardClasses } onClick={ this.onCardInteraction }>
```

Comment:
No joy with the bubbling for some reason. I wasn't seeing a `calypso_traintracks_interact` event fired before.